sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEArchitect.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Image,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import { Metric } from '../../../../components';
+
+export const HPEArchitect = ({ size }) => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  const columns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : ['3/4', 'auto'];
+  const gap = {
+    xsmall: 'medium',
+    small: 'medium',
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  };
+
+  const metricColumns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : { count: 3, size: 'auto' };
+
+  const areasDefault = [['quotes', 'hpeBook']];
+
+  const areasSmall = [
+    ['quotes', 'unassigned'],
+    ['hpeBook', 'unassigned'],
+  ];
+
+  const areas = ['xsmall', 'small'].includes(breakpoint)
+    ? areasSmall
+    : areasDefault;
+
+  return (
+    <PageContent
+      pad="large"
+      background={{
+        fill: 'horizontal',
+        // this is the closet color we have to what they have #00384D
+        color: 'graph-1',
+        dark: true,
+      }}
+    >
+      <Grid areas={areas} columns={columns} gap={gap[breakpoint]}>
+        <Box gap="xlarge" gridArea="quotes">
+          {/* // this heading is not large enough can we use size or we steering away? */}
+          <Box gap="medium">
+            <Heading size="xlarge" margin="none" level={2}>
+              Architect an AI advantage
+            </Heading>
+            <Text>
+              Overcoming overconfidence and oversight to help leaders secure
+              long-term business success.
+            </Text>
+          </Box>
+          <Box gap="large">
+            <Grid rows="auto" columns={metricColumns} gap="medium">
+              {/* we do not have a text size for these values that should be in 
+                the metric they should be larger  */}
+              <Metric
+                label="organizations have an official AI strategy in place"
+                value={
+                  <Text size="xxlarge" color="text-strong">
+                    9 in 10
+                  </Text>
+                }
+                reverse
+                options={null}
+                unit={null}
+                size={size}
+                alignSelf="start"
+                // the color here should be teal but is coming on dark teal we need light teal
+                border={{ side: 'left', color: '#82fff2', size: 'medium' }}
```

Comment:
We should avoid hard coding values. Instead, maybe we make it a box next to the metric? or find some other way to use the light version of teal?

Ideally the color used would be `decorative-teal`.

Reply:
Also detail on the design, but the "teal" bar should we the same height across all metrics, even if the content of one metric is shorter.

Reply:
Also detail on the design, but the "teal" bar should we the same height across all metrics, even if the content of one metric is shorter.

fixed the above ☝️

Reply:
not sure how I would get around using the light teal if the background is dark 🤔